Travel Scheduler/Views/GoToMoreOptionsCell.m

```diff
@@ -14,11 +14,20 @@
 static void instantiateImageView(UIImageView *imageView)
 {
     imageView.contentMode = UIViewContentModeScaleAspectFill;
-    imageView.layer.cornerRadius = 30;
+    imageView.layer.cornerRadius = 5;
     imageView.clipsToBounds = YES;
     [imageView.layer setBorderColor: [[UIColor whiteColor] CGColor]];
-    imageView.image = [UIImage imageNamed:@"iconfinder_104_Direction_Board_Camping_Sign_label_camping_camp_4172884"];
-    imageView.alpha = 0.5;
+    imageView.image = [UIImage imageNamed:@"2103641-512 (1)"];
```

Comment:
I think it might be helpful if you changed the image name after downloading it so we know which image it is you're adding here?

Reply:
Done!